qa/tasks/cephfs/test_misc.py

```diff
@@ -636,3 +636,75 @@ def test_alloc_cinode_assert(self):
 
         ls_out = set(self.mount_a.ls("test_alloc_ino/"))
         self.assertEqual(ls_out, set({"dir1", "dir2"}))
+
+
+class TestNewFSCreation(CephFSTestCase):
+    MDSS_REQUIRED = 1
+    TEST_FS = "test_fs"
+
+    """
+    This class aims to test the 'ceph fs new' cmd with new option 'set' that
+    enables to set fs options from 'ceph fs set'.
+    """
+
+    def test_fs_creation_valid_ops(self):
+        fs_ops = [["max_mds", "3"], ["refuse_client_session", "true"],
+                  ["allow_new_snaps", "true", "max_file_size", "65536"],
+                  ["session_timeout", "234", "session_autoclose",
```

Comment:
Add test with `--yes-i-really-mean-it`.

Reply:
`--yes-i-really-mean-it` is only used for `inline_data`, because it is deprecated, i decided not to add it, i'll add it to this test case

Reply:
Other settings in the future may add that. Best to have an existing test ensure it works.

Reply:
alright

Reply:
added